src/style.css

```diff
@@ -1 +1,48 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin:;
```

Comment:
It seems that the 'margin' property is empty for the '.stars' class. If you intended to reset the margin, specify '0'. If not, remove this property to clean up the code.